homeworks/roman.pavliuk_RomanPvlk/PokemonProject/style/pokemons.css

```diff
@@ -0,0 +1,98 @@
+/*eslint-disable*/
+
+.bottom-content .characteristic {
+  list-style-type: none;
+}
+
+.characteristic li {
+  font-size: 14px;
+  margin-bottom: 16px;
+  color: #221F1F;
+}
+
+.study_project {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.cards_section {
+  display: grid;
+  grid-template-rows: repeat(2, 1fr);
+  grid-template-columns: repeat(4, 1fr);
+  gap: 25px;
+  margin: 52px 0;
+}
+
+.card {
+  padding: 12px 12px 22px;
+  background-color: #ECECEC;
+  border-radius: 12px;
+}
+
+.title_section {
```

Comment:
Besides, you can see, how the tags touch one another on the small page width. You can fix the `min-width` of the card to prevent it
<img width="529" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/84c76a79-7030-4afe-952e-2b5742809d3b">